src/main/java/com/trivialTrivia/persistence/SessionFactoryProvider.java

```diff
@@ -0,0 +1,50 @@
+package com.trivialTrivia.persistence;
```

Comment:
Are we using this file at all? If not let's delete it

Reply:
NO I can take it out!

Reply:
Cool, I think it came out once you updated your branch.